internal/annotations/annotations_test.go

```diff
@@ -684,3 +684,251 @@ func TestExtractHostAliases(t *testing.T) {
 		})
 	}
 }
+
+func TestExtractConnectTimeout(t *testing.T) {
+	type args struct {
+		anns map[string]string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "empty",
+			want: "",
+		},
+		{
+			name: "non-empty",
+			args: args{
+				anns: map[string]string{
+					"konghq.com/connect-timeout": "3000",
+				},
+			},
+			want: "3000",
+		},
+	}
+	for _, tt := range tests {
```

Comment:
Out of scope for this--if we do want to make the loops within our tests parallel, we should do so in a separate PR. I don't think we'd need to, however--the unit test runtime is usually dwarfed by integration, and they don't run particularly slowly locally either. Most don't have enough cases where I'd expect the time savings to be that significant.

Reply:
I believe Tao did not mean  to make this run in parallel (I agree that there most likely is no point in this looking at potential time savings). I think the suggestion was just to capture the testcase struct to prevent this being shadowed by loop variable in case someone makes this parallel in the future.

Reply:
I do not want to make the test case to be run in parallel in this PR, but  capture the testcase struct to prevent this being shadowed by loop variable.